### PR TITLE
initial relase 0.6.23

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
python-pptx 0.6.23 ❄️ 
- [upstream](https://github.com/scanny/python-pptx/blob/v0.6.23/)
- [requirements](https://github.com/scanny/python-pptx/blob/v0.6.23/setup.py)
- updated recipe from conda forge
- satisfy linter
- added tests

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
+    - setuptools
   run:
-    - python =2.7|>=3.6
+    - python
     - lxml >=3.1.0
     - pillow >=3.3.2
-    - xlsxwriter >=0.5.7
-    - libiconv
-
+    - xlsxWriter >=0.5.7
```
Comment (Jrice1317) [inline recipe/meta.yaml:27]:
```suggestion
    - xlsxwriter >=0.5.7
```

**Review by SC426 — APPROVED** [review]
LGTM

**Review by Jrice1317 — COMMENTED** [review]
@M-Waszkiewicz-Anaconda Is flake8 included in another package or are we purposely excluding it? I see it listed [here](https://github.com/scanny/python-pptx/blob/v0.6.23/requirements.txt#L2) but not in the recipe

**Comment by M-Waszkiewicz-Anaconda** [comment]
> > @M-Waszkiewicz-Anaconda Is flake8 included in another package or are we purposely excluding it? I see it listed [here](https://github.com/scanny/python-pptx/blob/v0.6.23/requirements.txt#L2) but not in the recipe
> 
> Good catch!

flake8 checks the style and quality of some python code, is not needed in the package

**Comment by cbouss** [comment]
> here

Also, checking for requirements, what really matters is setup.py/pyproject.toml.
Here https://github.com/scanny/python-pptx/blob/v0.6.23/setup.py#L81 and here https://github.com/scanny/python-pptx/blob/v0.6.23/setup.py#L44.
Sometimes setup.py loads requirement.txt but not always.

**Review by cbouss — APPROVED** [review]

**Review by Jrice1317 — APPROVED** [review]

**Pull request merged**